README.md

```diff
@@ -1,151 +1,169 @@
-python-opensubtitles
-===
+# python-opensubtitles
 
 Simple module to access to the [OpenSubtitles.org](http://opensubtitles.org)
-subtitles database. This class is a wrapper to the common methods at OS.
+subtitles database. This class is a wrapper for the common methods used by the OpenSubtitles API.
 
-# Installing Notes
+## Install Notes
 
-If you are installing this using `pip`, please use the following format:
-`pip install -e git+https://github.com/agonzalezro/python-opensubtitles#egg=python-opensubtitles`
+If you are installing this module with `pip`, please use the following path to use the latest version:
+```
+pip install -e git+https://github.com/agonzalezro/python-opensubtitles#egg=python-opensubtitles
+```
 
-# Configuring the test environment
+## Configuring the test environment
 
-Before start to running the test (if you are only reading the documentation,
-of course, you don't need to do it :D) you must provide a correct video path
-and subtitle path.
+Before you start running tests (if you are only reading the documentation, of course, you don't need to do it :D), you must provide a correct video path and subtitle path.
 
-    >>> from os import path
-    >>> class Test(object):
-    ...     username = 'doctest'
-    ...     password = 'doctest'
-    ...     # Remember to change the video path if you want to run the test!
-    ...     name = 'Dark City'
-    ...     path = ('/home/alex/Videos/'
-    ...             'Dark.City.1998.Directors.Cut.BRRip.H264.AAC.5.1ch.Gopo/')
-    ...     video = 'Dark.City.1998.Directors.Cut.BRRip.H264.AAC.5.1ch.Gopo.mp4'
-    ...     subtitle = 'Dark.City.1998.Directors.Cut.BRRip.H264.AAC.5.1ch.Gopo.srt'
+```python
```

Comment:
I am not sure this will work with pydoctest. Did you try it?

Reply:
I didn't test anything because there aren't any tests included - to me, the testing instructions in the README read as incomplete example (as if they were started but never finished).

So, am I interpreting this correctly that everyone is supposed to test "manually" (with the code provided) instead of using a ready-made file/class? Wouldn't it make more sense to include that in the repo and then only have people set values for variables? (For their local file names.)

Reply:
Ohhh, wait. Having never used pydoctest, I thought the bit about testing was wrt a "regular" testing framework, when actually it's specifically for code examples in documentation? Huh, the more you know!

I'll have to read up on this.

Reply:
Hmm, ok, based on quickly skimming info on this, it looks like it won't work if it doesn't find `>>>` to denote examples for interactive Python sessions...

I never use the interpreter for anything but the simplest tests, definitely not for playing around with libraries, so find code examples given like that very impractical (bad for copy-pasting bits and pieces). But I don't know if this is maybe an unusual usage pattern.

Reply:
Those pydoctests will not work either because I am almost 100% sure that you don't have the film Dark City downloaded in: "Dark.City.1998.Directors.Cut.BRRip.H264.AAC.5.1ch.Gopo.mp4" :) That's why I mentioned on #18 that the tests should be moved and probably moved out from the README.

pydoctests is nice when you don't need a lot of setup for the tests, at the moment you need mocks I think it kinda miss its point.

Reply:
Yup, saw #18. Not sure what to do with this PR for now. Should the changes wait until other tests are implemented?

Reply:
@kerstin maybe do the changes you proposed but without touching the code snippets (you can change `os` for `ost` though). What do you think?

Reply:
Will revert the changes to the code snippets.

Reply:
hi @kerstin I agree that having documentation that is easy to copy/paste is a good idea; however, since we don't have proper tests yet don't change the formatting of the code here. We can do it after adding the tests somewhere else.